common/types/hashes.go

```diff
@@ -171,10 +171,22 @@ func (h Hash32) String() string {
 	return h.Hex()
 }
 
-// ShortString returns a the first 5 characters of the hash, for logging purposes.
+// MediumString returns the first 10 characters of the hash, for logging purposes.
```

Comment:
I don't mind having this, but I wonder what the motivation is... Have we seen cases where the 5-char `ShortString` is not enough and there were collisions?

Reply:
Yes, see the original motivation for this PR in #1570 - Barak was running into collisions with block id!

Reply:
Yeah, so let's make `ShortString()` return a 10-character string instead of 5.